package.json

```diff
@@ -169,6 +169,7 @@
   },
   "release": {
     "branches": [
+      "releases/+([0-9]).x",
```

Comment:
What kind of syntax is this? I read the docs and can't make sense of it. I'm not doubting it works, I just can't get my brain the tell me if I've seen this kind of thing before and how to decipher it.

Reply:
I don't know either. I tried putting it into regex and glob, and ran out of ideas shortly after.